miner/block_builder.go

```diff
@@ -398,6 +398,11 @@ func (t *BlockBuilder) handleGossipAtx(data service.GossipMessage) {
 		commitmentStr = atx.Commitment.String()
 	}
 
+	h, err := atx.NIPSTChallenge.Hash()
+	if err != nil {
```

Comment:
what happens if you try to print `h.string` below but you got an error here?

Reply:
empty string

Reply:
i take that back, fixed.